spec/csharp/.gitignore

```diff
@@ -0,0 +1,4 @@
+packages/
+.vs/
+bin/
+obj/
```

Comment:
1. It is generally a bad idea to omit newlines at the end of file. I'm not sure about C# style guide, but it's almost universally frowned upon in C/C++, Java, most scripting languages and in git itself.

2. Shouldn't we just put https://github.com/github/gitignore/blob/master/VisualStudio.gitignore there?